.github/workflows/release.yml

```diff
@@ -3,23 +3,56 @@ name: Release
 on:
     workflow_dispatch:
       inputs:
-        tag_name:
-          description: 'Proxy lib tag name + version'
+        proxy_lib:
+          description: 'Proxy lib name'
           required: true
+        proxy_lib_version:
+          description: 'Proxy lib version'
+          required: true  
+        min_version_supported:
+          description: 'Original package last supported version'
+          required: true    
 
 jobs:
+    test:
+        uses: ./.github/workflows/test.yml
+        secrets:
+          ACTIONS_GITHUB_TOKEN: ${{ secrets.ACTIONS_GITHUB_TOKEN }}
     tag:
-        needs: Test
         runs-on: ubuntu-latest
+        needs: [test]
         env:
           GH_ACCESS_TOKEN: ${{ secrets.ACTIONS_GITHUB_TOKEN }}
 
         steps:
           - name: Checkout Repo
             uses: actions/checkout@v3
 
-          - name: tag and push
-            uses: actions-ecosystem/action-push-tag@v1
+          - name: Tag and push
```

Comment:
Don't we validate first that version doesn't exist?

Reply:
it will fail here if it's already exists